crm_event/views/event_type_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<data>
+
+    <record id="view_event_type_form" model="ir.ui.view">
+        <field name="name">Link to events from type</field>
+        <field name="model">event.type</field>
+        <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'))]" />
+        <field name="inherit_id" ref="event.view_event_type_form"></field>
+        <field name="arch" type="xml">
+            <div position="before">
+                <!-- HACK Add .oe_inline to button box to avoid broken UI with
+                    other modules that also need to add the button box, such as
+                    event_sale_reservation -->
+                <div class="oe_button_box oe_inline" name="button_box">
+                    <!-- Use same iconography as in contact form -->
+                    <button colspan="2" class="oe_stat_button" type="action" name="%(event.action_event_view)d" icon="fa-ticket" context="{'default_event_type_id': id, 'search_default_event_type_id': id}">
```

Comment:
Can you switch this to a method for avoiding the problem with `id`?

Reply:
Done.